manifest/manifest.go

```diff
@@ -138,7 +149,11 @@ func (i *Input) parse() error {
 		i.Name = fmt.Sprintf("source:%s", i.Source)
 		return nil
 	}
-	return fmt.Errorf("input has an unknown type. Expect one, and only one of 'map', 'store' or 'source'")
+	if i.IsParams() {
+		i.Name = fmt.Sprintf("params:%s", i.Params)
```

Comment:
Shouldn't we validate here that only `string` is acceptable as a parameter? Unless it's done elsewhere, this should validate it.